zombies.js

```diff
@@ -79,6 +108,116 @@
  * @property {method} getMaxHealth         Returns private variable `maxHealth`.
  */
 
+class Player {
+
+  constructor(name, health, strength, speed) {
+    typeErrors(name, 'string');
+    typeErrors(health, 'number');
+    typeErrors(strength, 'number');
+    typeErrors(speed, 'number');
+    this.name = name;
+    this.health = health;
+    this.strength = strength;
+    this.speed = speed;
+    this._pack = [];
+    this._maxHealth = health;
+    this.isAlive = true;
+    this.equipped = false;
```

Comment:
`getPack` and `getMaxHealth` should be prototype methods and not defined in the constructor. The instructions may have made this confusing.